### PR TITLE
<feat>[calculadoraDay5]: calculadoraDay5.html added

### PR DESCRIPTION
**Pull request by axbsmurillor**
Tarea Dia 5, Básico JS

### CODE REVIEW

File: bogota-july-2018/javascript-calculator/Bryan Murillo/calculadora Day5.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        }
+    </script>
+</head>
+<body>
+    <header>
+        <h1>calculadora</h1>
+    </header>
+    <div>
+        <label for="a">a =</label>
+        <input type="number" name="number" id="a">
```
Comment (globantPipo) [inline bogota-july-2018/javascript-calculator/Bryan Murillo/calculadora Day5.html]: This is not a classic calculator please make an actual calculator UI

File: bogota-july-2018/javascript-calculator/Bryan Murillo/calculadora Day5.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            '*': function(a,b) {return a*b},
+            '/': function(a,b) {return b !== 0 ? a/b : "valor 0"}
+        }
+        
+        function calcular() {
+            let a = Number(document.getElementById("a").value);
+            let b = Number(document.getElementById("b").value);
+            
+            let opera = document.getElementById("operation").value
+            console.log(op[opera](a,b));
```
Comment (globantPipo) [inline bogota-july-2018/javascript-calculator/Bryan Murillo/calculadora Day5.html]: You are not even displaying in the UI the result

Comment (axbsmurillor) [reply bogota-july-2018/javascript-calculator/Bryan Murillo/calculadora Day5.html]: I'm sorry, this is not the UI calculator yet, this is the homework for Day 5, the UI calculator is the homework of Day 6.

Comment (globantPipo) [reply bogota-july-2018/javascript-calculator/Bryan Murillo/calculadora Day5.html]: Oh, i've assumed that with an html file with some UI

**Comment by axbsmurillor** [comment]
Thanks, I know I'm really late but I already make the pull request for the UI calculator, and I'm sorry for the confusion, the html was just to help me with the data inputs jeje

**Comment by globantPipo** [comment]
@axbsmurillor  Dont worry, im even more late than u checkign this PRs ;)

**Pull request merged**